### PR TITLE
Add introduction document of singleton python client sdk

### PR DESCRIPTION
**Pull request by jin-jasper**

### CODE REVIEW

File: content/en/docs/overview/singleton-sdk/python-client-introduction.md [hunk 1]
```diff
@@ -1,8 +1,202 @@
...
+```python
+class Translation:
+    def get_string(self, component, key, **kwargs)
+    def get_locale_strings(self, locale)
+    def get_locale_supported(self, locale)
+```
+
+### Translation / get_string
+
+* Get the trnalsation message of a component and a key.
```
Comment (Xiaochao8) [inline content/en/docs/overview/singleton-sdk/python-client-introduction.md:168]:
```suggestion
* Get the translation messages of a component and a key.
```

File: content/en/docs/overview/singleton-sdk/python-client-introduction.md [hunk 2]
```diff
@@ -1,8 +1,202 @@
...
+| Return | Description |
+| ------ | ------ |
+| object | Brief information data |
+
+
+## Interface For Translation
+### Translation
+```python
+class Translation:
+    def get_string(self, component, key, **kwargs)
```
Comment (Xiaochao8) [inline content/en/docs/overview/singleton-sdk/python-client-introduction.md:161]:
```suggestion
    def get_string(self, component, key, **args)
```

File: content/en/docs/overview/singleton-sdk/python-client-introduction.md [hunk 3]
```diff
@@ -1,8 +1,202 @@
...
 draft: false
-weight: 60
+weight: 10
 ---
 
-Coming soon
\ No newline at end of file
+
+# Introduction
+Several python files are provided as the python client to support accessing to Singleton service, making Singleton users to handle its globalization easily.
```
Comment (Xiaochao8) [inline content/en/docs/overview/singleton-sdk/python-client-introduction.md:10]:
```suggestion
Several python files are provided as the python client to support accessing Singleton service, making Singleton users to handle its globalization easily.
```

File: content/en/docs/overview/singleton-sdk/python-client-introduction.md [hunk 4]
```diff
@@ -1,8 +1,202 @@
...
+    def add_config_file(cls, config_file)
+    def add_config(cls, base_path, config_data)
+    def set_current_locale(cls, locale)
+    def get_current_locale(cls)
+    def get_release(cls, product, version)
+```
+
+### I18n / add_config_file
+
+* Load a configuration json or yaml file and initialize a correspondent release object.
```
Comment (Xiaochao8) [inline content/en/docs/overview/singleton-sdk/python-client-introduction.md:63]: TYPO: correspondent

File: content/en/docs/overview/singleton-sdk/python-client-introduction.md (hunk 1 above)
Comment (jin-jasper) [reply content/en/docs/overview/singleton-sdk/python-client-introduction.md:168]: Only one string is returned.

File: content/en/docs/overview/singleton-sdk/python-client-introduction.md (hunk 2 above)
Comment (jin-jasper) [reply content/en/docs/overview/singleton-sdk/python-client-introduction.md:161]: kwargs is a python style name that has fixed meaning.

File: content/en/docs/overview/singleton-sdk/python-client-introduction.md (hunk 4 above)
Comment (jin-jasper) [reply content/en/docs/overview/singleton-sdk/python-client-introduction.md:63]:
```suggestion
* Load a configuration json or yaml file and initialize a corresponding release object.
```

File: content/en/docs/overview/singleton-sdk/python-client-introduction.md (hunk 1 above)
Comment (Xiaochao8) [reply content/en/docs/overview/singleton-sdk/python-client-introduction.md:168]: It has a typo

File: content/en/docs/overview/singleton-sdk/python-client-introduction.md [hunk 5]
```diff
@@ -1,8 +1,202 @@
...
+```python
+class Translation:
+    def get_string(self, component, key, **kwargs)
+    def get_locale_strings(self, locale)
+    def get_locale_supported(self, locale)
+```
+
+### Translation / get_string
+
+* Get the trnalsation message of a component and a key.
```
Comment (jin-jasper) [inline content/en/docs/overview/singleton-sdk/python-client-introduction.md:168]:
```suggestion
* Get the translation message of a component and a key.
```

**Pull request merged**